debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorTask.java

```diff
@@ -97,7 +97,12 @@ public ChangeEventSourceCoordinator<MongoDbPartition, MongoDbOffsetContext> star
                     .loggingContextSupplier(() -> taskContext.configureLoggingContext(CONTEXT_NAME))
                     .build();
 
-            errorHandler = new MongoDbErrorHandler(connectorConfig, queue);
```

Comment:
@jcechace  How about using `errorHandler = new MongoDbErrorHandler(connectorConfig, queue, errorHandler);` ?

We could use the same approach for `SqlServerErrorHandler` too.